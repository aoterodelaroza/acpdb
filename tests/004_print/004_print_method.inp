## check: 004_print_method.out -a1e-10
## labels: regression quick

connect ../zz_source/bsip2-small/bsip2.db
print method
