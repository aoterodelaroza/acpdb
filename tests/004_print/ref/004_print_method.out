* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file ../zz_source/bsip2-small/bsip2.db
Connected database is sane

* PRINT: print the contents of the database 

| id| key| litrefs| description|
| 1| b3lyp/6-31+G*| bsip1 boys1970| This is a method.|
| 2| test| | |

